src/views/enterEmail.njk

```diff
@@ -0,0 +1,50 @@
+{% extends 'common/base.njk' %}
+{% set title = 'Enter Email' %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% block backLink %}
+  {{ govukBackLink({ text: "Back", href: "/choose-location" }) }}
+{% endblock %}
+{%block content%}
+
+  <form method="post"action="/validate-enter-email" id="enterEmailForm">
+    {{ govukInput({
+  label: {
+    text: "Enter your email address",
+    classes: "govuk-label--l",
+    isPageHeading: true
+  },
+  id: "enter-email",
+  name: "enterEmail",
+  errorMessage: {
+    text: "Enter your email address"
+  } if showError
+}) }}
+    {{ govukButton({
+  text: "Continue",
+  type: "submit"
+}) }}
+  </form>
+  <script>
+    document.addEventListener("DOMContentLoaded", function () {
+      const storedEmail = localStorage.getItem("enterEmail");
+      const textField = document.getElementById("enter-email");
+
+      textField.value = storedEmail ?? null;
+    });
+    document
+      .getElementById("enterEmailForm")
+      .addEventListener("submit", function (event) {
+        event.preventDefault();
+
+        const enterEmail = document
+          .getElementById("enter-email")
+          .value;
+
+        localStorage.setItem("enterEmail", enterEmail);
```

Comment:
maybe lets add a check so it only adds to the store if the value exists 

 ```
    if (enterEmail) {
          localStorage.setItem("enterEmail", enterEmail);
        }
```
<img width="1752" alt="Screenshot 2023-11-23 at 17 30 43" src="https://github.com/govuk-one-login/di-fec-ga4-demo/assets/38694448/5703e35e-c982-4c67-acb5-1591d4d8d986">

this will prevent adding to local storage if there's no value & the error messaging appears, i've added the same fix to the other form pages

Reply:
Also it would be good to add a regex check to see if there's a @ in there but, for the purposes of the demo app i'm lets leave it